__tests__/html/activityGrouping.groupingActivityStatus.html

```diff
@@ -187,15 +31,64 @@
 
         await pageConditions.webChatRendered();
 
-        const advanceButton = document.querySelector('.lolex__advance-button');
+        directLine.emulateIncomingActivity('Ullamco mollit sunt officia consectetur ex commodo.');
```

Comment:
It was not exactly correct to emulate outgoing activities by simply using `activity$` to dump things in.

To correctly emulate outgoing activities, it must go through Redux saga. This will capture logic such as send timeout.